src/Language/Hebrew/HebrewNounGenderInflector.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace NumberToWords\Language\Hebrew;
+
+class HebrewNounGenderInflector
+{
+    /**
```

Comment:
Could you please use type hinting instead of docblocks? This library requires min PHP7